pkg/blob/nodeserver.go

```diff
@@ -317,7 +317,7 @@ func (d *Driver) NodeStageVolume(ctx context.Context, req *csi.NodeStageVolumeRe
 		source := fmt.Sprintf("%s:/%s/%s", serverAddress, accountName, containerName)
 		mountOptions := util.JoinMountOptions(mountFlags, []string{"sec=sys,vers=3,nolock"})
 		if err := wait.PollImmediate(1*time.Second, 2*time.Minute, func() (bool, error) {
-			return true, d.mounter.MountSensitive(source, targetPath, NFS, mountOptions, []string{})
+			return true, d.mounter.MountSensitive(source, targetPath, AZNFS, mountOptions, []string{})
```

Comment:
this requires an upstream change in mount-utils?

https://github.com/kubernetes/kubernetes/blob/f61a0e41075c2c4468e6cda84f7a88a960fb2933/staging/src/k8s.io/mount-utils/mount_linux.go#L136

Reply:
Since we didn't set mounterPath: https://github.com/kubernetes-sigs/blob-csi-driver/blob/master/pkg/blob/blob.go#L265 for the mounter, I think it's not necessary to change mount-utils.